.circleci/config.yml

```diff
@@ -230,6 +230,17 @@ jobs:
     environment:
       BROWSER: << parameters.browser >>
       BVER: << parameters.bver >>
+    trigger-qe-tests:
+      docker:
+        - image: circleci/node:latest
+      steps:
+        - run:
+            name: Trigger QE tests
+            command: |
+              curl --silent --output /dev/null -u USER_TOKEN: -X POST \
```

Comment:
where does the USE_TOKEN come from? it does not seem to be one of the in-built or custom variables. Also don't you need $USER_TOKEN ?

Reply:
@DenissSazanovs You need to add a circle ci token to the `sdksqe` context with access to Video QE Slave repo. Please work with @ruescar to setup one.

Reply:
This is done already. Both variables are added

Reply:
SDKSQE_CIRCLECI_API_TOKEN has been added to sdks-qe context but won't work until sdksqe account is given Write permission on this project @syerrapragada

Reply:
@DenissSazanovs You should now be able to use the above variable

Reply:
sdksqe account has already been added to this project